android/canonical/app/src/main/java/com/google/samples/quickstart/canonical/RunFragment.kt

```diff
@@ -25,6 +25,21 @@ class RunFragment : Fragment() {
     // TODO: Rename and change types of parameters
     private var param1: String? = null
     private var param2: String? = null
+    private var pauseOffset = 0L
+    private var isWorking = false
```

Comment:
I think these would fit well in a ViewModel that isn't directly tied to this instance of the Fragment. https://developer.android.com/topic/libraries/architecture/viewmodel

Also consider how this could this survive the app being killed in the background for resources then resumed later. For example you could store state in onPause and restore in onResume so that users could start the timer without worrying about the app dying.